File: src/BreakfastProvider.Api/Models/Responses/AuditLogResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public record AuditLogResponse
{
    public Guid AuditLogId { get; init; }
    public string Action { get; init; } = string.Empty;
    public string EntityType { get; init; } = string.Empty;
    public Guid EntityId { get; init; }
    public string Details { get; init; } = string.Empty;
    public DateTime Timestamp { get; init; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/AuditLogResponse.java
package io.lemonlion.breakfast.model.response;

import java.time.Instant;
import java.util.UUID;

/** Twin of C# {@code AuditLogResponse}. */
public record AuditLogResponse(
        UUID auditLogId, String action, String entityType, UUID entityId, String details, Instant timestamp) {
}
